virttest/utils_test/__init__.py

```diff
@@ -433,6 +433,12 @@ def find_python(session, try_binaries=('python3', 'python2', 'python')):
         if session.cmd_status("which %s" % python) == 0:
             return python
 
+def find_host_python(try_binaries=('python3', 'python2', 'python')):
+    for python in try_binaries:
+        out = process.run("which %s" % python, shell=True)
```

Comment:
You need `ignore_status=True` here, otherwise it will raise error on the host without python3.